internal/service/tracker_service.go

```diff
@@ -95,7 +95,7 @@ func (t *TrackerService) Commit(
 				return 0, err
 			}
 
-			snapshot := store.Snapshot{Object: newObject.Object}
+			snapshot := store.Snapshot{Object: newObject.Object, Time: time.Now()}
```

Comment:
let's create a newSnapshot function with Object and Time as required parameters

Reply:
done in a6eb3d27696e